### PR TITLE
Add Windows Forms/.NET 4.6 sample.

### PR DESCRIPTION
**Pull request by anurse**
image

### CODE REVIEW

**Comment by anurse** [comment]
Derp. Yes. I did indeed re-title it though

**Comment by anurse** [comment]
I also tested this with our latest nightly build and it works as well, but I didn't want to check that in to this repo.

File: WindowsFormsSample/WindowsFormsSample/ChatForm.cs [hunk 1]
```diff
@@ -0,0 +1,132 @@
...
+        }
+
+        private void OnSend(string message)
+        {
+            Log(Color.Black, message);
+        }
+
+        private void Log(Color color, string message)
+        {
+            messagesList.Items.Add(new LogMessage(color, message));
```
Comment (davidfowl) [inline WindowsFormsSample/WindowsFormsSample/ChatForm.cs]: Uhh @anurse how does this work when you're not on the sync context?

Comment (JamesNK) [reply WindowsFormsSample/WindowsFormsSample/ChatForm.cs]: That's a good point. OnSend should call Log in WinForms dispatcher

Comment (davidfowl) [reply WindowsFormsSample/WindowsFormsSample/ChatForm.cs]: I want to do more thinking about this. Here was the original discussion https://github.com/SignalR/SignalR/issues/972

Comment (anurse) [reply WindowsFormsSample/WindowsFormsSample/ChatForm.cs]: Well it works perfectly... I can update the the sample to put it on the dispatcher though.

Comment (anurse) [reply WindowsFormsSample/WindowsFormsSample/ChatForm.cs]: It doesn't throw and that was good enough for me at this point.

**Pull request merged**